kotlin/add-graphics-renderer/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.esri.arcgisruntime.sample.addgraphicsrenderer">
+
+    <uses-permission android:name="android.permission.INTERNET" />
+
```

Comment:
missing the glESversion requirement here

`    <uses-feature
            android:glEsVersion="0x00020000"
            android:required="true"/>`